src/app/pages/FeatureIdxPage/index.test.jsx

```diff
@@ -0,0 +1,242 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import fetchMock from 'fetch-mock';
+import { BrowserRouter } from 'react-router-dom';
+import { render, act } from '@testing-library/react';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { ToggleContextProvider } from '#contexts/ToggleContext';
+import afriqueFeatureIdxPageData from '#data/afrique/cpsAssets/48465371';
+import afriqueMostReadData from '#data/afrique/mostRead';
+import getInitialData from '#app/routes/cpsAsset/getInitialData';
+import FeatureIdxPage from '.';
+
+const mockToggles = {
+  ads: {
+    enabled: false,
+  },
+  mostRead: {
+    enabled: true,
+  },
```

Comment:
Let's remove both of these because we're not displaying Ads or MostRead on FIX for the moment